ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/infrastructure/LedgerTestContext.scala

```diff
@@ -63,4 +65,9 @@ final class LedgerTestContext(
   def semanticTesterLedger(parties: Set[Ref.Party], packages: Map[Ref.PackageId, Ast.Package]) =
     new SemanticTesterLedger(bindings)(parties, packages)(this)
 
+  def extractErrorStatus[A](future: Future[A]): Future[Status.Code] =
+    future.failed
+      .collect {
+        case NonFatal(ex: StatusRuntimeException) => ex.getStatus.getCode
```

Comment:
Should we also catch [`io.grpc.StatusException`](https://grpc.github.io/grpc-java/javadoc/io/grpc/StatusException.html)s?

Reply:
Addressed by acb912d.